.github/workflows/push.yml

```
@@ -105,3 +105,19 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+  licensed:
+    name: License Checker
+    runs-on: [ ubuntu-latest ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      - uses: jonabc/setup-licensed@v1
+        with:
+         version: 3.x
+
+      - run: go get
+
```

```suggestion
      - run: go get
```

Do we need these spaces?